src/tests/ftest/server/metadata.py

```diff
@@ -62,10 +61,14 @@ class ObjectMetadata(TestWithServers):
     """
 
     # Minimum number of containers that should be able to be created
-    CREATED_CONTAINERS_MIN = 2900
+    CREATED_CONTAINERS_MIN = 25000
+    # would be lower with engine env_vars DAOS_MD_CAP=128
+    # CREATED_CONTAINERS_MIN = 6500
 
     # Number of created containers that should not be possible
-    CREATED_CONTAINERS_LIMIT = 7500
+    CREATED_CONTAINERS_LIMIT = 36000
```

Comment:
Same as above. Please add created container limit var to metadata.yaml.

Reply:
added this in the latest code on the branch.